compiled-templates/compiled-templates.js

```diff
@@ -1,65 +0,0 @@
-module.exports.render = function (options) {
```

Comment:
I was able to completely replace the other way we had of building static pages from a .md file with build-static-html.